src/pages/AdoptablePetsDetails.jsx

```diff
@@ -21,16 +22,20 @@ const AdoptablePetsDetails = props => {
       <div className="container">
         <div className="row">
           <div className="col-md-8 col-sm-12">
-            <div id="dg_main-content">
-              <PetDetail
-                imageUrl={imageUrl}
-                imageUrlAltText={imageUrlAltText}
-                aboutMe={aboutMe}
-                attributes={attributes}
-                animalName={animalName}
-                id={id}
-              />
-            </div>
+            {isLoading ? (
+              <p>Loading Animal Services Pets...</p>
```

Comment:
this naming should probably be consistent with the list page.

Reply:
done